src/app/domain/backup/models/Backup.ts

```diff
@@ -0,0 +1,46 @@
+import { Media, BackupedMedia } from '/app/domain/backup/models'
+
+type BackupStatus = 'to do' | 'initializing' | 'ready' | 'running' | 'done'
```

Comment:
I love the confidence that comes from not being afraid to use space!

Reply:
😅

Changed to 'to_do' just in case 😄